### DISCUSSION TITLE
Many thanks, but a problem

### DISCUSSION THREAD

**Discussion by mik55** [johang/sd-card-images#52] [Q&A]
First, many thanks for the images. Great work!
But i have a problem. I have a tinkerboard and will use a ssd connected drive over an usb adapter as root partition. In Armbian ist this possible (Armbian-config Option 1 in System).  
Is there a way for this? 
Thank you and regards
Michael

**Comment by johang** [comment]
I'm not sure how Armbian does it, but as far as I know Rockchip doesn't support booting from USB.

**Reply by mik55** [reply]
Hello johang, 
yes i know this restriction. 
But my intention is only the root filesystem on the partition (sda1 ) from my ssd. The boot  partition must remain on the SD-card. 
I make a lot of trails with the file "cmdline.txt". My idea ist to change the config from u-boot. But where?
thanks
Michael

**Reply by johang** [reply]
But my SD card images have only one partition. Root filesystem (/) and boot (/boot) is on the same partition.

**Comment by mik55** [comment]
Hello Johang,
thats possible. 
many regardsMichael







    Am Montag, 22. Mai 2023 um 22:26:03 MESZ hat Johan Gunnarsson ***@***.***> Folgendes geschrieben:  
 
 


But my SD card images have only one partition. Root filesystem (/) and boot (/boot) is on the same partition.

**Comment by mik55** [comment]
Thanks a lot for your help!
regards 
Michael
